src/wasm/encoder.h

```diff
@@ -59,6 +59,8 @@ class WasmFunctionBuilder : public ZoneObject {
                const uint32_t* local_indices,
                uint32_t indices_size);
   void AppendCode(const byte opcode, bool add_local_operand);
+  uint32_t AppendEditableCode(const byte opcode);
```

Comment:
This method is used with the immediate to an opcode, so it should probably be AppendEditableImmediate(), and the parameter name likewise updated.